packages/core/src/components/forms/_common.scss

```diff
@@ -15,14 +15,14 @@ $input-color-disabled: $button-color-disabled !default;
 $input-placeholder-color: $pt-text-color-muted !default;
 $input-background-color: $white !default;
 $input-background-color-disabled: rgba($light-gray1, 0.5) !default;
-$input-shadow-color-focus: $pt-intent-primary !default;
+$input-shadow-color-focus: $pt-focus-indicator-color;
```

Comment:
if this can't be relied on externally - remove and use `$pt-focus-indicator-color` directly in the handful of places it's used. Same for dark theme version below.

Reply:
it looks like we explicitly distribute `_colors.scss`, `_color-aliases` and `_variables`, but not this file. could consider updating to use `$pt-focus-indicator-color` directly

`"dist:variables": "generate-css-variables --retainDefault true ../../colors/src/_colors.scss common/_color-aliases.scss common/_variables.scss",`